conf_files/log.yaml

```diff
@@ -7,8 +7,7 @@ logger:
         datefmt: '%H:%M:%S'
       detail:
         style: '{'
-        # See FilenameLineFilter in logger.py for fileline description
-        format: '{levelname:.1s}{asctime}.{msecs:03.0f} {fileline:20s} {message}'
+        format: '{levelname:.1s}{asctime}.{msecs:03.0f} {filename} {lineno} {message}'
```

Comment:
I'd prefer we still have a fixed length for `filename` and `lineno` so the the `message` always start with a left flush. I could live without but I think it makes reading easier.

Reply:
Done.